ansible/roles/common/tasks/main.yml

```diff
@@ -21,6 +21,13 @@
   retries: 10
   delay: 15
   until: epelresult is not failed
+# Update ca-certificates to ensure that CA certificates are new enough
```

Comment:
I wonder if this is needed as part of the `common` role, given we usually do not directly download anything directly on the nodes (or maybe we do)?

Reply:
I was thinking that at first too, but I searched around for all `get_url` calls. There are definitely some (e.g., in ansible/roles/elasticsearch/tasks/main.yml as one example) that are invoked on the target host directly and not just the proxy.